change/@fluentui-react-accordion-ec93c77c-698f-4e0d-960f-b047c7791b6f.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
+  "comment": "chore: update Griffel package",
```

Comment:
Should we go with something more human like "Update Griffel to latest version"?

Reply:
Makes sense, done 43de50c 👍